spec/features/admin_update_and_delete_superpowers_spec.rb

```diff
@@ -24,10 +23,10 @@
     expect(page).to have_content("Superpower successfully updated")
   end
 
-  scenario 'user should only see edit link if they added the superpower' do
+  scenario 'only admin should see edit link' do
     user2 = FactoryBot.create(:user)
     sign_in_as(user2)
-    user1 = FactoryBot.create(:user)
+    user1 = FactoryBot.create(:user, role: 'admin')
```

Comment:
I'd really like to clean this up by declaring `user1` above `user2` (in other words, just re-ordering your lines)